src/api-engine/api/routes/chaincode/views.py

```diff
@@ -177,7 +177,7 @@ def install(self, request):
         try:
             cc_targz = ""
             file_path = os.path.join(FABRIC_CHAINCODE_STORE, chaincode_id)
-            for _, _, files in os.walk(file_path):
+            for _, _, files in os.walk secure_file_path(file_path):
```

Comment:
There are a couple of issues here.  Have reverted this PR.
1. Syntax error here
2. `secure_file_path` function wasn't imported.